source/includes/openshift/deploy-minio-on-openshift.rst

```diff
@@ -136,7 +136,92 @@ The web console displays a widget for tracking the installation progress.
 
 Once installation completes, click :guilabel:`View Operator` to view the MinIO Operator page. 
 
-3) Open the MinIO Operator Interface
+3) Configure TLS Certificates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have installed the MinIO Operator from Red Hat OperatorHub, the installation process also configures the :openshift-docs:`OpenShift Service CA Operator <security/certificate_types_descriptions/service-ca-certificates.html>`.
+This Operator manages the the TLS certificate required to access the MinIO Operator Console and Tenants.
+It automatically renews and rotates the certificate 13 months before expiration.
+No additional action is required.
+
+For Operator installations deployed from Red Hat Marketplace or other methods, configure the :openshift-docs:`Service CA certificates <security/certificate_types_descriptions/service-ca-certificates.html>` manually.
+See the instructions in the dropdown below for details.
+
+.. dropdown:: OpenShift Service CA Certificate configuration
+
+   To manually enable the ``service-ca`` Operator to manage TLS certificates:
+
+   #. Configure the following MinIO :ref:`environment variables <minio-server-environment-variables>`:
+
+      .. list-table::
+         :header-rows: 1
+
+         * - Environment Variable
+           - Value
+
+         * - :envvar:`MINIO_CONSOLE_TLS_ENABLE`
+           - ``on``
+
+         * - :envvar:`OPERATOR_STS_ENABLED`
+           - ``off``
+
+         * - :envvar:`MINIO_OPERATOR_RUNTIME`
+           - ``OpenShift``
+
+   #. Configure the following ``volumes`` and ``volumeMounts`` in ``something.yaml``:
```

Comment:
I'm unclear where this YAML is updated

Reply:
This is in the Operator deployment. Celis has a YAML file to use with `oc apply` that will do this for you, no editing needed (for "standard" environments)

Reply:
Added example of editing the file. Can dig into the `oc apply` method on a future revisit. (Unclear where that YAML file should live.)